### PR TITLE
Mute failing test see #30456

### PR DESCRIPTION
**Pull request by alpar-t**
Mute test due to #30456

### CODE REVIEW

File: x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/UpgradeClusterClientYamlTestSuiteIT.java [hunk 1]
```diff
@@ -29,6 +30,7 @@
 import static org.hamcrest.Matchers.is;
 
 @TimeoutSuite(millis = 5 * TimeUnits.MINUTE) // to account for slow as hell VMs
+@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/30456")
```
Comment (nik9000) [inline x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/UpgradeClusterClientYamlTestSuiteIT.java]: We usually directly import `AwaitsFix`.

**Review by nik9000 — APPROVED** [review]
LGTM. For these things we usually just push the fix without a PR in an effort to get the build green.

We should also figure out why the CLA checker bot doesn't like you.

**Comment by davidkyle** [comment]
@atorok when you merge please backport this to 6.x as the tests also fail on that branch

**Comment by nik9000** [comment]
I wouldn't normally do this but I'm going to merge this myself rather than wait for @atorok to get to it in the morning because the build is pretty busted and I'd like to give us a chance to merge other things before ml folks can take a look at this.

I'm going to ignore all the checks too ignore all of the failures because the ci ones look like some kind of strange git issue and the CLA one is just out of date. I know that @atorok is ok CLA wise.

**Comment by jasontedor** [comment]
@nik9000 @atorok Add `.patch` to the PR URL and notice that there is a typo in the commit email address.

**Comment by atorok** [comment]
Thanks @nik9000! @davidkyle  I checked and this is in 6.x as well so probably nick did both.

**Comment by jtibshirani** [comment]
Just cherry-picked this to 6.3, as we've just seen the same failures there: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.3+multijob-unix-compatibility/os=centos/54/console

**Pull request merged**